usaspending_api/common/helpers/decorators.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from django.db import connection
+from django.utils.decorators import method_decorator
+
+DEFAULT_DB_TIMEOUT_IN_MS = 300000
+logger = logging.getLogger(__name__)
+
+
+def set_db_timeout(timeout_in_ms=DEFAULT_DB_TIMEOUT_IN_MS):
```

Comment:
Can this value be in seconds instead of ms? The value can be multiplied by 1000 when it is setting `statement_timeout` in postgres

Reply:
I can change that